CoreScriptsRoot/Modules/Settings/Utility.lua

```diff
@@ -313,6 +317,17 @@ local function isGuiVisible(gui, debug) -- true if any part of the gui is visibl
 	end
 end
 
+local function addHoverState(button, instance, applyNormal, applyHover)
+	local function indirectNormal()	applyNormal(instance) end
```

Comment:
This is true. The intent with "indirectNormal" was that the function provided indirection and can be used for event connections. I can't think of a better name for it...